xtask/src/commands/all.rs

```diff
@@ -10,6 +10,7 @@ use super::Test;
 pub struct All {
     #[structopt(flatten)]
     compliance: Compliance,
+
```

Comment:
```suggestion
```

Reply:
I could not get `cargo fmt` to run reliably on `xtask/` stuff.